[2024-12-30T05:01:50.264+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2024-12-30T05:01:44.555084+00:00 [queued]>
[2024-12-30T05:01:50.268+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2024-12-30T05:01:44.555084+00:00 [queued]>
[2024-12-30T05:01:50.268+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-12-30T05:01:50.269+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-12-30T05:01:50.269+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-12-30T05:01:50.274+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2024-12-30 05:01:44.555084+00:00
[2024-12-30T05:01:50.279+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2024-12-30T05:01:44.555084+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp86lag_p4']
[2024-12-30T05:01:50.280+0000] {standard_task_runner.py:83} INFO - Job 11: Subtask store_user
[2024-12-30T05:01:50.277+0000] {standard_task_runner.py:55} INFO - Started process 240 to run task
[2024-12-30T05:01:50.306+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user manual__2024-12-30T05:01:44.555084+00:00 [running]> on host 8de73b8ac67f
[2024-12-30T05:01:50.332+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2024-12-30T05:01:44.555084+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-30T05:01:44.555084+00:00
[2024-12-30T05:01:50.333+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2024-12-30T05:01:50.336+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-12-30T05:01:50.338+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-12-30T05:01:50.342+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20241230T050144, start_date=20241230T050150, end_date=20241230T050150
[2024-12-30T05:01:50.379+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-12-30T05:01:50.388+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
